src/Symfony/Bridge/Doctrine/ManagerRegistry.php

```diff
@@ -45,34 +43,6 @@ protected function resetService($name): void
 
             return;
```

Comment:
Isn't the `if ($manager instanceof LazyObjectInterface) {` above also dead code ? Or is there still a case where we use a LazyObjectInterface of var-exporter instead of a native lazy object on PHP 8.4+ ?